### PR TITLE
[Accuracy diff No.89] Fix accuracy diff for paddle.incubate.nn.functional.fused_bias_act API

### PR DESCRIPTION
**Pull request by hushenwei2000**
现有 `paddle_to_torch` 的 `paddle.incubate.nn.functional.fused_bias_act` API 实现有错误，已修复。在 fp16 精度测试下需要设置 `atol = 1`（因为在量化时涉及 `round()` 操作会四舍五入）。

`tester/api_config/5_accuracy/accuracy_gpu_error.txt` 中的 20 个错误 case 现在可以通过。

### CODE REVIEW

**Comment by cangtianhuang** [comment]
能定位是 torch 算错了还是 paddle 算错了呀？127 很大了

File: tester/api_config/5_accuracy/accuracy_gpu_error.txt [hunk 1]
```diff
@@ -2280,26 +2280,6 @@ paddle.einsum("mixy,bmxy,kmxy->bixy", Tensor([32, 32, 1, 1],"float32"), Tensor([
 paddle.gammaln(Tensor([10, 20, 1],"float32"), )
 paddle.gammaln(Tensor([2, 3, 4, 5],"float32"), )
 paddle.gammaln(Tensor([2, 3, 4, 5],"float64"), )
-paddle.incubate.nn.functional.fused_bias_act(Tensor([2, 22016],"int32"), None, act_method="swiglu", compute_dtype="fp16", dequant_scales=Tensor([22016],"float32"), shift=None, smooth=None, quant_scale=0.0009313154732808471, quant_round_type=0, quant_max_bound=127.0, quant_min_bound=-127.0, )
-paddle.incubate.nn.functional.fused_bias_act(Tensor([2, 22016],"int32"), None, act_method="swiglu", compute_dtype="fp16", dequant_scales=Tensor([22016],"float32"), shift=None, smooth=None, quant_scale=0.0009654839523136616, quant_round_type=0, quant_max_bound=127.0, quant_min_bound=-127.0, )
```
Comment (wanghuancoder) [inline tester/api_config/5_accuracy/accuracy_gpu_error.txt:2284]: 这些case不需要删除，还在这个文件里放着就可以。这里放着：
1. 可以用于未来小范围的回归测试
2. 如果所有问题都修复后，可以改变文件名。但这些case都是宝贵的测试用例还是需要保留下来的。

Comment (hushenwei2000) [reply tester/api_config/5_accuracy/accuracy_gpu_error.txt]: 好的，已更新第二个 commit 恢复了这些文件

File: tester/paddle_to_torch/rules.py [hunk 2]
```diff
@@ -1917,17 +1920,25 @@ def geglu(x):
...
-        x = x / quant_scale
-        if quant_round_type == 0:
-            x = torch.round(x)  # Round to nearest, ties to even
-        elif quant_round_type == 1:
-            x = torch.where(x >= 0, torch.ceil(x - 0.5), torch.floor(x + 0.5))
-        else:
-            raise ValueError(f"Unsupported quant_round_type: {quant_round_type}")
-        x = x * quant_scale
-        x = torch.clamp(x, min=quant_min_bound, max=quant_max_bound)
+        x = quant_helper_func(x, quant_scale, quant_round_type, quant_max_bound, quant_min_bound)
```
Comment (cangtianhuang) [inline tester/paddle_to_torch/rules.py:1937]: 可以直接将 quant_helper_func 在这展开吗？这样代码更清晰一点

File: tester/paddle_to_torch/rules.py [hunk 3]
```diff
@@ -1917,17 +1920,25 @@ def geglu(x):
...
+        print("after quant", x)
```
Comment (cangtianhuang) [inline tester/paddle_to_torch/rules.py:1938]: 删去

File: tester/paddle_to_torch/rules.py [hunk 4]
```diff
@@ -1917,17 +1920,25 @@ def geglu(x):
...
+        
+        x = x.to(getattr(torch, "int8"))
```
Comment (cangtianhuang) [inline tester/paddle_to_torch/rules.py:1940]:
```suggestion
        x = x.to(torch.int8)
```

**Review by cangtianhuang — CHANGES_REQUESTED** [review]

**Review by cangtianhuang — APPROVED** [review]
LGTM
